File: e002.c
#include <stdio.h>
#include <math.h>

int main(void)
{
    int a, b, c, somme, max;
    a = b = c = 1;
    somme = 0;
    max = 4 * pow(10, 6);
    while (c <= max)
    {
        if (c % 2 == 0)
        {
            somme += c;
        }
        c = a + b;
        a = b;
        b = c;
    }

    printf("%i", somme);

}

File: e002.java
class Main
{
    public static void main(String[] args)
    {
        int a, b, c;
        a = b = c = 1;
        
        int total, limit;
        limit = 4 * (int) Math.pow(10, 6);
        total = 0;

        while (c <= limit)
        {
            if (c % 2 == 0)
            {
                total += c;
            }
            c = a + b;
            a = b;
            b = c;
        }
        System.out.print(total);
    }
}
